docs/config.json

```
@@ -621,6 +621,10 @@
             {
               "title": "Dynamic Registration",
               "slug": "/application-access/guides/dynamic-registration/"
+            },
+            {
+              "title": "AWS DynamoDB",
```

Is there any concern about having the same title for 2 different pages? Will it make searching hard, for example?

---

You're not wrong. How about:

```suggestion
              "title": "AWS DynamoDB as an Application",
```

---

This value only affects how the page is displayed in the sidebar, where it will be nestled under the Application menu. I haven't reviewed our search index policy, but I'm fairly confident that it uses the page title defined on the page itself, which is different.